.travis.yml

```diff
@@ -10,16 +10,18 @@ jobs:
     # Latest image build/push
     - if: branch = master
       script:
+        - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
         - make latest && make test
       deploy:
         provider: script
         script: echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin && make push-latest
```

Comment:
For compatibility with PRs from forks, you'll need to conditionally login.
https://docs.travis-ci.com/user/pull-requests/#pull-requests-and-security-restrictions